hack/lib/golang.sh

```diff
@@ -433,6 +433,44 @@ kube::golang::get_physmem() {
   echo 1
 }
 
+
+kube::golang::verify_code_builds() {
+  if ! OUT=$(which gb 2>&1); then
+    hack/build-go.sh
```

Comment:
Worth printing something like "if you had github.com/constabulary/gb installed, this might be faster" ?